tracee/event_monitor_ebpf.c

```diff
@@ -1724,8 +1724,13 @@ int send_bin(struct pt_regs *ctx)
     chunk_size = bin_args->full_size - i*F_CHUNK_SIZE;
 
     if (chunk_size > F_CHUNK_SIZE) {
-        bin_args->full_size = chunk_size;
-        bin_args_map.update(&id, bin_args);
+        bin_args_t bin_args_new = {};
```

Comment:
```suggestion
        // create a copy of the map before submitting - workaround for an issue with old kernels
        bin_args_t bin_args_new = {};
```

Reply:
It actually the same as it was before a previous commit - nothing odd here